src/realizations/catchment/Tshirt_C_Realization.cpp

```diff
@@ -37,13 +39,12 @@ Tshirt_C_Realization::Tshirt_C_Realization(forcing_params forcing_config,
                                            std::vector<double> giuh_ordinates,
                                            tshirt::tshirt_params params,
                                            const vector<double> &nash_storage)
-        : Catchment_Formulation(catchment_id, std::move(std::make_unique<Forcing>(forcing_config)), output_stream), catchment_id(std::move(catchment_id)),
+        //: Catchment_Formulation(catchment_id, std::move(std::make_unique<Forcing>(forcing_config)), output_stream), catchment_id(std::move(catchment_id)),
+        : Catchment_Formulation(catchment_id, std::move(std::make_unique<CsvPerFeatureForcingProvider>(forcing_config)), output_stream), catchment_id(std::move(catchment_id)),
```

Comment:
I'm betting this should not be specific to the CSV reader... where is this constructor used? Maybe this is only in test code and that's okay, but not sure.

Reply:
It looks like the class is use by a few testing codes. Not sure which form of the constructors.